modules/ml/pom.xml

```diff
@@ -87,6 +87,12 @@
         </dependency>
 
         <dependency>
+            <groupId>com.github.fommil.netlib</groupId>
+            <artifactId>core</artifactId>
+            <version>1.1.2</version>
```

Comment:
> Also I doubt about classes like IgniteIntDoubleToDoubleBiFunction.
This is done to avoid unnecessary autoboxing.
> extract version as param
Done.